CMSgov/mint-app
# Easi 2686/user table migration cleanup

# EASI-2686

## Changes and Description

- Remove old baseStruct and baseTaskList
    - Rename baseStructUserTable to baseStruct
    - Rename baseTaskListUserTable to baseTaskList
- Removed redundant created by and modified by account fetching logic
   - implemented embedded createdByUserAccount and modifiedByUserAccount in BaseStruct and ExistingModel

- Remove IT tools from front end and backend

- Remove python migration script

- Remove unused EUA generation tools


## How to test this change
1. Make sure unit tests pass
2. Make sure that you can run the app
3. Look through the code for any other instances of old EUAIDs that can be removed from the app.


## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
